doc/hooks/zig.section.md

```diff
@@ -0,0 +1,51 @@
+# zigHook {#zighook}
+
+[Zig](https://ziglang.org/) is a general-purpose programming language and toolchain for maintaining robust, optimal and reusable software.
+
+In Nixpkgs, `zigHook` overrides the default build, check and install phases.
+
+## Example code snippet {#example-code-snippet}
+
+```nix
+{ lib
+, stdenv
+/* . . . */
+, zigHook
+}:
+
+stdenv.mkDerivation {
+/* . . . */
+
+   nativeBuildInputs = [
+     zigHook
+   ];
+
+/* . . . */
+}
```

Comment:
maybe something like this could be better

```suggestion
{ lib
, stdenv
, zigHook
}:

stdenv.mkDerivation {
  # ...

  nativeBuildInputs = [
    zigHook
  ];
  
  zigBuildFlags = [ "-Dman-pages" ];

  dontUseZigCheck = true;

  # ...
}
```